src/gui/mainwindow.cpp

```diff
@@ -1454,8 +1454,11 @@ void MainWindow::loadPreferences(const bool configureSession)
 
     showStatusBar(pref->isStatusbarDisplayed());
 
-    if ((pref->preventFromSuspendWhenDownloading() || pref->preventFromSuspendWhenSeeding()) && !m_preventTimer->isActive()) {
-        m_preventTimer->start(PREVENT_SUSPEND_INTERVAL);
+    if (pref->preventFromSuspendWhenDownloading() || pref->preventFromSuspendWhenSeeding()) {
```

Comment:
Sorry for resurrecting this, but I can't understand what changed here. Aren't if conditions the same in the old and new code? Why break them up?

Reply:
>Aren't if conditions the same in the old and new code? 

`else` isn't the same.

Reply:
Damn. I didn't realize that!

Reply:
This is elusive at first glance...